src/stylesheets/_forms_actions.scss

```diff
@@ -2,8 +2,7 @@
     flex: 0 0 auto;
     padding: 2.4rem 0 0;
     display: flex;
-    flex-wrap: wrap;
-    flex-direction: row-reverse;
+    flex-flow: row-reverse wrap;
```

Comment:
I'm curious why here we're using consolidated declarations and in other places we're expending them... I think it was a transition shorthand that got expanded into transition-duration and what not. Also above we use the shorthand for flex.